packages/api/src/external/hie/shared.ts

```diff
@@ -18,6 +21,13 @@ export async function createOrUpdateInCq(
   coordinates: Coordinates
 ): Promise<void> {
   const { log } = out("createOrUpdateInCq");
+  const orgExistsInDirectory = await doesOrganizationExistInCQ(cxOrg.oid);
+
+  if (!orgExistsInDirectory) {
+    log(`Organization ${cxOrg.name} with OID ${cxOrg.oid} does not exist in the CQ directory`);
```

Comment:
Is this sufficient feedback? If you register a facility using `PUT /internal/facility`, and the execution goes here, logs and returns, this will still not notify the user that the operation was a failure, right? Don't we need to throw an error and provide elaborate feedback?

Reply:
We dont await this function so the best we can do is log this. The idea is whoever runs this internal endpoint should validate it works and if it doesnt then check the logs to see what went wrong.

Reply:
Shouldn't we just execute those synchronously? It's not like this takes a long time to execute + this is only used in this one internal route. I think this will simplify things for us to know immediately what went wrong, if anything